clients/admin-ui/src/features/config-wizard/DescribeSystemsForm.tsx

```diff
@@ -0,0 +1,209 @@
+  // not just use this default
+  const { data: existingSystem } = useGetSystemByFidesKeyQuery(
+    "default_organization"
+  );
+
+  const toast = useToast();
+
+  // TODO FUTURE: is key stored? If so, where does it exist in the system API?
+  const initialValues = {
+    name: existingSystem?.name ?? "",
+    description: existingSystem?.description ?? "",
+    system_dependencies: existingSystem?.system_dependencies ?? [],
+    system_type: existingSystem?.system_type ?? "",
+  };
+
+  const handleSubmit = async (values: FormValues) => {
+    const systemBody = {
+      fides_key: existingSystem?.fides_key ?? "default_organization",
+      name: values.name ?? existingSystem?.name,
+      description: values.description ?? existingSystem?.description,
+      privacy_declarations: existingSystem?.privacy_declarations ?? [
+        {
+          name: "string",
+          data_categories: ["string"],
+          data_use: "string",
+          data_qualifier:
+            "aggregated.anonymized.unlinked_pseudonymized.pseudonymized.identified",
+          data_subjects: ["string"],
+          dataset_references: ["string"],
+        },
+      ],
+      system_type: values.system_type ?? existingSystem?.system_type,
+      system_dependencies:
+        values.system_dependencies ?? existingSystem?.system_dependencies,
+      // TODO: tags = system_dependencies?
+    };
+
+    setIsLoading(true);
+
+    if (!existingSystem) {
+      // @ts-ignore
+      const { error: createSystemError } = await createSystem(systemBody);
+
+      if (createSystemError) {
+        toast({
+          status: "error",
+          description: "Creating system failed.",
+        });
+      } else {
+        toast.closeAll();
+        handleChangeStep(5);
+      }
+    } else {
+      // @ts-ignore
+      const { error: updateSystemError } = await updateSystem(systemBody);
+
+      if (updateSystemError) {
+        toast({
+          status: "error",
+          description: "Updating system failed.",
+        });
+      } else {
+        toast.closeAll();
+        handleChangeStep(5);
+      }
+    }
+
+    setIsLoading(false);
+  };
+
+  return (
+    <Formik initialValues={initialValues} onSubmit={handleSubmit}>
+      <Form>
+        <Stack ml="100px" spacing={10}>
+          <HorizontalStepper activeStep={1} steps={HORIZONTALSTEPS} />
+          <Heading as="h3" size="lg">
+            {/* TODO FUTURE: Path when describing system from infra scanning */}
+            Describe your system
+          </Heading>
+          <div>
+            By providing a small amount of additional context for each system we
+            can make reporting and understanding our tech stack much easier for
+            everyone from engineering to legal teams. So let’s do this now.
+          </div>
+          <Stack>
+            <Stack direction="row">
+              <CustomTextInput name="name" label="System name" />
```

Comment:
This will be really interesting for later when I have to match the designs more closely where a question mark icon exists, but it doesn't really generate a tooltip...it generates text on a separate part of the page that isn't a tooltip (I think we paired on this briefly when we were trying to problem-solve the mouseover stuff). 

So probably won't want to pass a tooltip prop to these inputs in the future and instead have the question mark there as  its own separate component with a hover trigger that displays text on the other half of the page.

If you're fine with it for now, since the question mark icon will need to exist on its own regardless in the future, I think we can keep it like this rather than passing the tooltip prop when #776  is merged in?